M/MPICH/build_tarballs.jl

```diff
@@ -110,6 +125,7 @@ augment_platform_block = """
 
 platforms = supported_platforms()
```

Comment:
```suggestion
platforms = supported_platforms(; exclude=x->Sys.iswindows(x))
```

Reply:
https://github.com/JuliaPackaging/Yggdrasil/blob/5de2d55fad6f061323986dc2e2ee6a22911b2ee5/M/MPICH/build_tarballs.jl#L129 it's already there (otherwise it wouldn't work in the first place)?

Reply:
Yep. Just cleans up the syntax.

Reply:
I'm using the `filter!` syntax in many other projects, including MicrosoftMPI, OpenMPI, and MPItrampoline. It's easier to use when several unrelated platforms need to be disabled.